### DISCUSSION TITLE
Flashback on iPad

### DISCUSSION THREAD

**Discussion by Renad-L** [thebaselab/codeapp#703] [Q&A]
I update the app in new version on my iPad, then it always have flashback problems when I open the app.
Usually I have to click the app three or more times to open the app.
My iPadOS version is 15.7.1 update to 15.7.2, still have this problem.

**Comment by bummoblizard** [comment]
Thank you for reporting this. We are looking into it and will fix it as soon as possible. Did you try to restart your iPad?

**Comment by bummoblizard** [comment]
Are you able to provide the crash log to me?:
- Open the Analytics & Improvements section of Settings on the device. See Share analytics, diagnostics, and usage information with Apple.
- Tap Analytics Data.
- Locate the log for your app. The log name starts with <AppBinaryName>_<DateTime> for crash reports, or JetsamEvent_<DateTime> for high-memory use crashes.
- Select the desired log.
- Tap the Share icon, and select Mail to send the crash report as a mail attachment.

My email is sh.chung@thebaselab.com.
